hack/development-mode.sh

```diff
@@ -15,6 +15,10 @@ $ROOT/hack/util-update-app-of-apps.sh $MY_GIT_REPO_URL development $MY_GIT_BRANC
 # this needs to be pushed to your fork to be seen by argocd
 $ROOT/hack/util-set-development-repos.sh $MY_GIT_REPO_URL development $MY_GIT_BRANCH
 
+# set the API server which SPI uses to authenticate users to empty string (by default) so that multi-cluster
```

Comment:
does this need to be added to `preview.sh`?

Reply:
Good point, it's been added.